### PR TITLE
Finalizing data-search script & cron job

### PR DESCRIPTION
**Pull request by Suhyma**
Just added some code to format the final JSON after the search is completed. The properties of the JSON elements now match the output of the original search.sh script.

Refs #22

### CODE REVIEW

File: src/cli.js [hunk 1]
```diff
@@ -29,9 +29,20 @@ export async function search (queryString, options) {
       combineWith: options.strict ? 'AND' : 'OR'
     });
 
-    await sendOutput(res, options, queryString);
-
-    return res;
+    // format res before giving final output JSON
+    const formattedRes = res.map(article => ({
```
Comment (jvwong) [inline src/cli.js:33]: This mapping fixes the data input and output schema, making the function less reusable (e.g. what if the input article schema changes such as fetching from another organizations). An alternative is to pass a formatting function.

**Review by jvwong — APPROVED** [review]

**Comment by maxkfranz** [comment]
@Suhyma @jvwong 

There's a conflict in the PR that would need to be addressed in both the PR and @Suhyma's branch going forward.

**Review by jvwong — CHANGES_REQUESTED** [review]
Great - just wondering about the file open/close.

File: src/data-search.js [hunk 2]
```diff
@@ -1,61 +1,52 @@
...
+    const { keywords } = config;
+    const papers = await searcher.search(keywords, {
+      combineWith: 'AND'
+    });
+    return _.assign({}, config, { papers });
+  };
+  const collection = await Promise.all(config.map(doSearches));
+
+  // Output all search result papers into data.json
+  const fileHandle = await fs.openSync('example-data/data.json', 'w');
```
[the remark this answers is not in the document]
Comment (Suhyma) [reply src/data-search.js]: You're right, just removed the extra file open and close.

**Review by jvwong — APPROVED** [review]
Works for me.

**Pull request merged**